ProcessMaker/Cache/Settings/SettingCacheManager.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+namespace ProcessMaker\Cache\Settings;
+
+use Illuminate\Cache\CacheManager;
+use ProcessMaker\Cache\CacheInterface;
+
+class SettingCacheManager implements CacheInterface
+{
+    protected CacheManager $cacheManager;
+
+    public function __construct(CacheManager $cacheManager)
+    {
+        $this->cacheManager = $cacheManager;
+    }
+
+    /**
+     * Dynamically pass method calls to the cache manager.
+     *
+     * @param string $method
+     * @param array $arguments
+     * @return mixed
+     */
+    public function __call($method, $arguments): mixed
+    {
+        return $this->cacheManager->$method(...$arguments);
+    }
+
+    /**
+     * Get a value from the settings cache.
+     *
+     * @param string $key
+     * @param mixed $default
+     *
+     * @return mixed
+     */
+    public function get(string $key, mixed $default = null): mixed
+    {
+        return $this->cacheManager->get($key, $default);
```

Comment:
improve with error handling example
   ```
try {
    return $this->cache->get($key, $default);
       } catch (\Exception $e) {
           // Log the error
           \Log::error("Cache get error: " . $e->getMessage());
           return $default;
       }
```

Reply:
Error handling was added

Reply:
By the way, there is a ticket [FOUR-20909](https://processmaker.atlassian.net/browse/FOUR-20909) to go deeper into it.